### ISSUE TITLE
JavaScript Developer Tools: Introduce this page after OOP intro

### ISSUE THREAD

**Issue by rawfiul** [TheOdinProject/curriculum#26912]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

I understand we want students to learn debugging tools as soon as they learn conditionals and data types, but all the articles and videos on this page are too early to digest.
I suggest either a or b:
A - a custom-made precise tutorial on this page itself where students are introduced to tools and files.js that just debugs conditionals for now.
B - teach students about debugging/introduce this current page, after a small intro to object oriented aspect of js.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-javascript-developer-tools

### (Optional) Discord Name

rawfiul

**Comment by JoshDevHub** [comment]
Hello @rawfiul 

Can you specifically point to the place(s) in the articles where you think the use of object orientation is confusing?

**Comment by JoshDevHub** [comment]
Bumping this @rawfiul 

Would be very glad to discuss specific issues you felt were difficult to work through due to lack of knowledge around objects or object orientation.

**Comment by rawfiul** [comment]
hey sorry, i have been busy with exams. Will write a detailed report and
revert back

**Comment by JoshDevHub** [comment]
@rawfiul Thanks for updating me. No need to rush, just wanted to make sure you're still interested.

**Comment by JoshDevHub** [comment]
Any update on this @rawfiul ?

**Comment by TheeMee** [comment]
I agree with this since I also struggled to follow through all the linked articled and documents. It refers to concepts which are uncovered before in the course. Such as animations, caught and uncaught exception among others. I especially struggled with DOM manipulation with node selectors and CSS at-rules.

**Comment by JoshDevHub** [comment]
I'm going to close this because of no activity, but if anyone still feels strongly about it, definitely tag me here and we can talk more about it.
